hw/top_earlgrey/sw/autogen/top_earlgrey_memory.h

```diff
@@ -913,6 +913,15 @@
  * `TOP_EARLGREY_RV_CORE_IBEX_CFG_BASE_ADDR + TOP_EARLGREY_RV_CORE_IBEX_CFG_SIZE_BYTES`.
  */
 #define TOP_EARLGREY_RV_CORE_IBEX_CFG_SIZE_BYTES 0x1000
+
+/**
+ * MMIO Region
+ *
+ * MMIO region excludes ROM, RAM, and flash but includes retention RAM.
+ */
+#define TOP_EARLGREY_MMIO_BASE_ADDR 0x40000000u
```

Comment:
just curious, do we need to have this in both `top_earlgrey.h` and `top_earlgrey_memory.h`?

Reply:
Yes, because we need these defines both in C and asm and we can only include `top_earlgrey_memory.h` in asm.

Reply:
ah thanks for explaining.